source/server/http/admin.h

```diff
@@ -162,6 +159,16 @@ class AdminImpl : public Admin,
   void addListenerToHandler(Network::ConnectionHandler* handler) override;
   Server::Instance& server() { return server_; }
 
+  using CallbackFunction = std::function<Http::Code(
```

Comment:
reference `AdminFilter::CallbackFunction` rather than duplicating the definition here.

Reply:
Sorry, my bad! I have updated the code.